data/wanderer/wanderers middle.txt

```diff
@@ -3488,6 +3489,7 @@ event "wanderers: sestor planets unrestricted"
 	planet "Sopi Lefarkata"
 		"required reputation" 0
 		spaceport clear
+		description `This is a cold and mountainous world, and most of its oceans are covered in ice. A handful of giant craters gouged out from the sides of mountains are the only sign that there were once Korath settlements here. But, the Kor Sestor robots have begun building settlements of their own, in caverns deep under the mountains where they are safe from orbital bombardment. Where the final set of Kor Mereti attacks collapsed one of the access tunnels, the swarms of robots that gathered to dig them out now sit motionless, slowly being buried under snow and dust.`
```

Comment:
The idea with the factories being shut down seems to have been that the robots are no longer being replaced. The finite supply of already manufactured machines is destroyed and there are no more to replace them. I'm not sure how much sense it makes for these robots to have just gone inactive like this.

Reply:
Yeah, that's a fair point. I'll drop that change then, and just keep the Sestor Ikfar one.

Reply:
It might make sense for those robots to have scattered or just be gone because they've been destroyed and no new ones have arrived.

Reply:
The Mereti would also have stopped attacking, I imagine.